qa/test-old-engine/pom.xml

```diff
@@ -80,7 +80,7 @@
             <groupId>org.camunda.bpm</groupId>
             <artifactId>camunda-bom</artifactId>
             <!-- Cannot use variables due to bug in Maven Release Plugin -->
-            <version>7.22.0</version>
+            <version>7.23.0</version>
             <scope>import</scope>
             <type>pom</type>
           </dependency>
```

Comment:
Sometimes the dependencies here are updated based on the changes to `engine/pom.xml`
The only related changes since last minor: 
- [To remove mariaDB](https://github.com/camunda/camunda-bpm-platform/commit/b877f105577f3209fb1320cad3b8a9f08407d3d5#diff-9f0112d180a7258b7ab81f65ca090e031a715e51c0b001aa52659974047bd13b)
- [bump Spring to v6](https://github.com/camunda/camunda-bpm-platform/commit/c8374f26a28434f103f8f6cc95d7b2ac20af29ac#diff-9f0112d180a7258b7ab81f65ca090e031a715e51c0b001aa52659974047bd13b)